### PR TITLE
[ja] Update Japanese translation

### PR DESCRIPTION
**Pull request by mkismy**
Fixed machine translation.

### CODE REVIEW

File: locales/ja/validation-inline.php [hunk 1]
```diff
@@ -12,116 +12,116 @@
...
     'ipv4'                 => '有効なIPv4アドレスである必要があります。',
     'ipv6'                 => '有効なIPv6アドレスである必要があります。',
-    'json'                 => 'これは有効なJSON文字列である必要があります。',
+    'json'                 => '有効なJSON文字列である必要があります。',
     'lt'                   => [
-        'array'   => 'The content must have less than :value items.',
-        'file'    => 'The file size must be less than :value kilobytes.',
-        'numeric' => 'The value must be less than :value.',
-        'string'  => 'The string must be less than :value characters.',
+        'array'   => ':value 個より少ない必要があります。'
```
Comment (caouecs) [inline locales/ja/validation-inline.php:68]: need ','

**Review by caouecs — CHANGES_REQUESTED** [review]
Hello,
thank you, just a fix to do

File: locales/ja/validation-inline.php (hunk 1 above)
Comment (mkismy) [reply locales/ja/validation-inline.php]: Sorry for that. Fixed in 34fa86dcf9ba74bd4052826fdc230521a8a376bd

**Comment by caouecs** [comment]
thank you

**Comment by caouecs** [comment]
@mkismy Can you translate the last 6 tags ? https://github.com/Laravel-Lang/lang/blob/master/docs/statuses/ja.md

Thanks by advance

**Pull request merged**